app/src/androidTest/java/ch/epfl/qedit/quiz/QuizActivityTest.java

```diff
@@ -138,6 +138,37 @@ public void testTimeIsClicked() {
         finishActivity();
     }
 
+    @Test
```

Comment:
I think that we should add some actual checks to the tests. For example, check that the confirmation dialog is shown when clicking on the validate button.

Reply:
Ok I just realized this is covered by another test of the same class, so I guess it is ok.